ingestion/setup.py

```diff
@@ -43,6 +43,9 @@ def get_long_description():
     "requests~=2.26",
     "cryptography",
     "PyYAML",
+    "opentelemetry-exporter-otlp-proto-http>=1.6.2,<1.7",
```

Comment:
any reason for this to be less than 1.7

Reply:
This has been ported from previous Pull request. Not sure if we need to update this.
Ref - https://github.com/open-metadata/OpenMetadata/pull/1397/files